### PR TITLE
Add a Fork Me Block to the home page

### LINKED ISSUE #712: Display "demo" notice on sample sites
Display a prominent demo notice that links to this git repo.

- https://hoverboard-master.firebaseapp.com/
- https://hoverboard-dev.firebaseapp.com/

### PR DESCRIPTION
**Pull request by cortinico**
I've drafted a _Fork Me Block_ to have in the home page.
I haven't extract strings or icons as I don't expect users to use it (or do we?).
Also I'm happy to change the wording to something else if needed.

Screenshot 2020-02-25 at 00 32 16

Closes #712

### CODE REVIEW

**Comment by abraham** [comment]
This is great!

- I don't think the strings needs to be extracted. I wouldn't expect any other site to run it but the examples.
- The white text is probably not accessible. It'll need more color contrast.
- It would be nice if this would automatically be excluded on non-example sites. The approach I would look at would be to add a key/value to `settings.json` of something like `"showForkMeBlockOnProjectIds": ["hoverboard-master", "hoverboard-dev"]` and then wrap the `<fork-me-block></fork-me-block>` HTML and the import in nunjucks template if the current `firebase.projectId` isn't in that array.

**Comment by cortinico** [comment]
> and then wrap the <fork-me-block></fork-me-block> HTML and the import in nunjucks template if the current firebase.projectId isn't in that array.

Done.
I was just not able to wrap the `import` in the if statement. Looks like `nunjucks` tags are processed only for the body. Am I missing something 🤔

**Comment by abraham** [comment]
`nunjucks` should apply to all `json`, `html`, `js`, and maybe `md` files. I'll take a look tonight though.

**Comment by abraham** [comment]
I'm not a super big fan of this but it does work. It required a change to eslint and would need to be polyfilled for older Edge. The component is small enough that I think we can just always bundle it for now.

```js
if (JSON.parse('{% if showForkMeBlockForProjectIds.includes(firebase.projectId) %}true{% else %}false{% endif %}')) {
  import('../elements/fork-me-block.js');
}
```

File: src/elements/fork-me-block.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      }
+
+    </style>
+
+    <div class="container container-narrow">
+      <h1 class="container-title">Fork me on GitHub</h1>
+      <p>
+        Hoverboard is open source conference website template and is developed entirely on a voluntary basis.
+        You can check the source code that generated this website on Github.
+        If you find a issue or you want to contribute, you're more than welcome!
```
Comment (abraham) [inline src/elements/fork-me-block.js]: @pearlbea How does this copy cound?

Comment (pearlbea) [reply src/elements/fork-me-block.js]: 👍

**Pull request merged**